config/production.json

```diff
@@ -47,7 +47,7 @@
 		"google-my-business": true,
 		"happychat": true,
 		"help": true,
-		"help/gpt-response": false,
+		"help/gpt-response": true,
```

Comment:
Copying this here: https://github.com/Automattic/wp-calypso/pull/75838/files#r1187604756

Reply:
Yep, understood. We'll test just in caplyso first.